src/utils/internal/getCallFrame.test.ts

```diff
@@ -0,0 +1,163 @@
+import { getCallFrame } from './getCallFrame'
+
+const mockStack = jest.fn()
+jest.spyOn(window, 'Error').mockImplementation(() => ({
+  name: 'mockError',
+  message: '',
+  stack: mockStack(),
+}))
+
+test('Node on Linux and macOS error stack', () => {
+  // version 1
+  mockStack.mockImplementationOnce(() =>
+    [
+      'Error: ',
+      '    at getCallFrame (/Users/mock/github/msw/lib/umd/index.js:3735:22)',
+      '    at Object.get (/Users/mock/github/msw/lib/umd/index.js:3776:29)',
+      '    at Object.<anonymous> (/Users/mock/github/msw/test/msw-api/setup-server/printHandlers.test.ts:13:8)', // <-- this one
+      '    at Runtime._execModule (/Users/mock/github/msw/node_modules/jest-runtime/build/index.js:1299:24)',
+      '    at Runtime._loadModule (/Users/mock/github/msw/node_modules/jest-runtime/build/index.js:898:12)',
+      '    at Runtime.requireModule (/Users/mock/github/msw/node_modules/jest-runtime/build/index.js:746:10)',
+      '    at jasmine2 (/Users/mock/github/msw/node_modules/jest-jasmine2/build/index.js:230:13)',
+      '    at runTestInternal (/Users/mock/github/msw/node_modules/jest-runner/build/runTest.js:380:22)',
+      '    at runTest (/Users/mock/github/msw/node_modules/jest-runner/build/runTest.js:472:34)',
+    ].join('\n'),
+  )
+
+  expect(getCallFrame()).toBe(
+    '/Users/mock/github/msw/test/msw-api/setup-server/printHandlers.test.ts:13:8',
+  )
+
+  // version 2
+  mockStack.mockImplementationOnce(() =>
+    [
+      'Error: ',
+      '    at getCallFrame (/Users/mock/git/msw/lib/umd/index.js:3735:22)',
+      '    at graphQLRequestHandler (/Users/mock/git/msw/lib/umd/index.js:7071:25)',
+      '    at Object.query (/Users/mock/git/msw/lib/umd/index.js:7182:18)',
+      '    at Object.<anonymous> (/Users/mock/git/msw/test/msw-api/setup-server/printHandlers.test.ts:14:11)', // <-- this one
+      '    at Runtime._execModule (/Users/mock/git/msw/node_modules/jest-runtime/build/index.js:1299:24)',
+      '    at Runtime._loadModule (/Users/mock/git/msw/node_modules/jest-runtime/build/index.js:898:12)',
+      '    at Runtime.requireModule (/Users/mock/git/msw/node_modules/jest-runtime/build/index.js:746:10)',
+      '    at jasmine2 (/Users/mock/git/msw/node_modules/jest-jasmine2/build/index.js:230:13)',
+      '    at runTestInternal (/Users/mock/git/msw/node_modules/jest-runner/build/runTest.js:380:22)',
+      '    at runTest (/Users/mock/git/msw/node_modules/jest-runner/build/runTest.js:472:34)',
+    ].join('\n'),
+  )
+
+  expect(getCallFrame()).toBe(
+    '/Users/mock/git/msw/test/msw-api/setup-server/printHandlers.test.ts:14:11',
+  )
+})
+
+test('Node on Windows error stack', () => {
```

Comment:
Do you mind grouping these test scenarios under their respective OS?

```js
describe('MacOS', () => {
  // ...
})

describe('Windows', () => {
  // ...
//
```

I agree that flat tests are good, but in this case grouping them would help scanning trough quicker.

Reply:
- [ ] group tests by OS

Reply:
hm,, what do you want me to do with `Chrome and Edge error stack` and `Firefox on macOS and Windows error stack` tests please? they have the same error stack on both

Reply:
Oh, a good point. I haven't noticed some of the test suites shared the stack trace across OS. I don't mind, let's keep it as-is.